src/core/etl/src/Flow/ETL/Row/Entry/ArrayEntry.php

```diff
@@ -76,6 +78,11 @@ public function name() : string
         return $this->name;
     }
 
+    public function phpType() : Type
+    {
+        return new ArrayType([] === $this->value);
```

Comment:
it would be more efficient to store phpType as class property created in constructor